### DISCUSSION TITLE
Timelapse render not working.

### DISCUSSION THREAD

**Discussion by YKcinY** [Guilouz/Creality-Helper-Script-Wiki#449] [Issues]
Hey, 

Today I updated everything that had an update available. After that the Timelapse Render does not work anymore. 
I am a noob when it comes to klipper so I might be at fault here updating.

**Comment by spitfire** [comment]
I've had this too. There's a bug in Moonraker Timelapse https://github.com/mainsail-crew/moonraker-timelapse/issues/158. It might be related to https://github.com/mainsail-crew/moonraker-timelapse/issues/126. Check Moonraker.log

**Comment by Galloant** [comment]
I had the same issue for quite a while. What I didn't realize is that I had `TIMELAPSE_TAKE_FRAME` missing in my g-code on layer change. Without that, the script doesn't get any input to take a frame.

See this article for details: https://github.com/mainsail-crew/moonraker-timelapse/blob/main/docs/configuration.md
